lib/resque/data_store.rb

```diff
@@ -57,7 +57,11 @@ def initialize(redis)
     # Compatibility with any non-Resque classes that were using Resque.redis as a way to access Redis
     def method_missing(sym,*args,&block)
       # TODO: deprecation warning?
-      @redis.send(sym,*args,&block)
+      if sym == :select
+        @redis.select(args.first)
```

Comment:
I don't understand why you can't send the `select` method that same way as all others.